examples/oneapi/cpp/source/knn/knn_kd_tree_batch.cpp

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
```

Comment:
It is not very consistent with another classifier - SVM. Let's either align according to your example or mine.

Reply:
Do we need to add train_model/test_model functions? Don't like it.

Reply:
No, it doesn't need. We did not add either to the svm or the DF.